src/components/safe-apps/AppCard/styles.module.css

```diff
@@ -12,3 +12,22 @@
   top: 2px;
   left: 2px;
 }
+
+.truncateDescription {
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```

Comment:
Please don't use browser-specific hacks. It should work in all browsers.

Reply:
We have the same styles [in our old codebase](https://github.com/safe-global/safe-react/blob/dev/src/routes/safe/components/Apps/components/SafeAppCard/SafeAppCard.tsx#L217) for this and it seems that it has a good browser support (`97.47%`):

```
https://caniuse.com/?search=-webkit-line-clamp
```

What is your opinion about it?

Reply:
Prefixed CSS rules are by definition browser-specific/temporary.
Did you test in Firefox?
It might work (for now), but we should in general avoid them and stick to the standard CSS spec.

Reply:
it seems that[ only the prefixed css rule is available](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp).

For me it's working fine in Chrome, Firefox and Safari.

```
https://css-tricks.com/line-clampin/#aa-the-standardized-way
```